content/en/docs/contributing/release-process.md

```diff
@@ -68,25 +81,38 @@ The process for cutting a minor release is as follows:
     git push --set-upstream origin
     ```
 
-4. [Create release notes](#release-notes)
+    > ⚠️ You need to be an "admin" to be able to push to `release-1.0`.
 
-    Sanity check the notes, checking that the notes contain details of all the features and bug fixes that you expect to be in the release.
+4. [Generate and edit the release notes](#generating-and-editing-the-release-notes)
 
 5. Run `cmrel stage`
 
-    In this example we stage a release using the 'release-1.0' branch,
-    setting  the release version to `v1.0.0`:
+   1. In this example we stage a release using the 'release-1.0' branch,
+      setting  the release version to `v1.0.0`:
 
-    ```bash
-    cmrel stage --branch=release-1.0 --release-version=v1.0.0
-    ```
+      ```bash
+      cmrel stage --branch=release-1.0 --release-version=v1.0.0
+      ```
+
+      This step takes ~10 minutes. It will build all Docker images and
+      create all the manifest files and upload them to a storage bucket on
+      Google Cloud. These artifacts will be published and released in the
+      next steps.
+
+      The final line of output contains URL of the bucket
+      containing the release artifacts. The final segment in that URL
+      contains the `RELEASE_NAME`, which you will need in the next step.
 
-    This step takes ~10 minutes.
-    It will build all Docker images and create all the manifest files and upload them to a storage bucket on Google Cloud.
-    These artifacts will be published / released in the next steps.
+   2. While the build is running, send a first Slack message to
+      `#cert-manager-dev`:
 
-    The final line of output contains URL of the bucket containing the release artifacts.
-    The final segment in that URL contains the `RELEASE_NAME`, which you will need in the next step.
+      > Releasing 1.2.0-alpha.2 🧵
+
+   3. Send a second Slack message in reply to this first message with the
```

Comment:
Would it be worth to add here a warning to check at this point whether the Cloud Build output doesn't contain any sensitive data?